### DISCUSSION TITLE
Anti-Kick

### DISCUSSION THREAD

**Discussion by Gloktober** [winstxnhdw/lc-hax#88] [Ideas]
Hey ! Would it be possible to create something that would unable the host to kick you ? Also , an in-game name changer would be nice !

**Comment by winstxnhdw** [comment]
These are great suggestions. I have tried to block the host from kicking before, but I couldn't get it to work. It may very well be server-sided.

A name changer may be possible but looking at the game code, it seems that other players will only read your name once you join the lobby and never again. This means if there's such a feature, you would have to re-join the lobby for it to work.

**Reply by Gloktober** [reply]
I see ! Thanks for the answer :D
